suite-common/wallet-core/src/fees/feesThunks.ts

```diff
@@ -0,0 +1,184 @@
+            const result = await TrezorConnect.blockchainEstimateFee({
+                coin: network.symbol,
+                request: {
+                    blocks: [2],
+                    feeLevels: 'smart',
+                    specific: {
+                        from: '0x0000000000000000000000000000000000000000',
+                        to: '0x0000000000000000000000000000000000000000',
+                    },
+                },
+            });
+            if (result.success) {
+                const feeLevelBase = result.payload.levels[0];
+                const isEip1559ActivatedAndAvailable =
+                    getNetwork(network.symbol).features.includes('eip1559') &&
+                    isEip1559(feeLevelBase) &&
+                    isDebugModeActive &&
+                    !device?.unavailableCapabilities?.['eip1559'];
+
+                if (isEip1559ActivatedAndAvailable) {
+                    newFeeInfo = result.payload;
+                } else {
+                    newFeeInfo = {
+                        ...result.payload,
+                        levels: [
+                            {
+                                ...feeLevelBase,
+                                baseFeePerGas: undefined,
+                                maxFeePerGas: undefined,
+                                maxPriorityFeePerGas: undefined,
+                                maxWaitTimeEstimate: undefined,
+                                label: 'normal' as const,
+                            },
+                        ],
+                    };
+                }
+            }
+        } else {
+            const result = await TrezorConnect.blockchainEstimateFee({
+                coin: network.symbol,
+                request: {
+                    feeLevels: 'smart',
+                },
+            });
+            if (result.success) {
+                newFeeInfo = {
+                    ...result.payload,
+                    levels: sortLevels(
+                        result.payload.levels
+                            // hack to hide "low" fee option
+                            // (we do not want to change the connect API as it is a potentially breaking change)
+                            .filter(level => level.label !== 'low'),
+                    ),
+                };
+            }
+        }
+
+        if (newFeeInfo) {
+            const partial: Partial<NetworksFees> = {};
+            partial[network.symbol] = {
+                blockHeight: blockchainInfo.blockHeight,
+                ...newFeeInfo,
+            };
+
+            dispatch(feesActions.updateFee(partial));
+        }
+    },
+);
+
+export const removeFeeInfoThunk = createThunk(
+    `${FEES_MODULE_PREFIX}/removeFeeInfoThunk`,
+    ({ networks }: { networks: NetworkSymbol[] }, { dispatch, getState }) => {
+        const fees = selectFees(getState());
+        const activeFees = Object.keys(fees) as NetworkSymbol[];
```

Comment:
nit: something like this would be better
```suggestion
        const networksWithFees = Object.keys(fees) as NetworkSymbol[];
```